src/utils/hooks/useDevPassword.js

```diff
@@ -8,7 +8,10 @@ export const useDevPassword = () => {
 
     const get = () => sessionStorage.getItem(devPasswordKey)
 
-    const getEncoded = () => encodeBase64(sessionStorage.getItem(devPasswordKey))
+    const getEncoded = () => {
+        const password = get()
+        return password ? encodeBase64(password) : ''
+    }
 
     return {get, set, getEncoded}
```

Comment:
Nie wiem czy takie nazwy w słowniku są najlepsze XD

Reply:
Ale teraz zmieniać je wszędzie to będzie porażka XDDD

Reply:
Znaczy ogólnie możesz je zmienić podczas dekonstrukcji obiektu, nikt Ci nie zabroni, jakbyś potrzebował innych nazw. Ale ogólnie wydaje mi się, że na razie jest to bez sensu zmiana teraz tego wszędzie XD

Reply:
Znaczy rozumiem że zmiana tego teraz wszędzie będzie strasznie męcząca. Ale z drugiej strony gdzieś funkcji get czy set nie za wiele mówi co ta funkcja zrobi. Dodatkowo można by rozważyć dodanie funkcji którą pisałem wyżej getDevPassword. Albo zamiana po prostu funkcji getEncoded na getPassword. XDD

Reply:
Może shift+f6 ogarnie zmianę sygnatur, chociaż w sumie wątpię, że potrafi w destrukcję obiektu